releases/borg-1.2.rst

```diff
@@ -0,0 +1,84 @@
+Borg 1.2
+========
+
+`« back to all releases <.>`_
+
+:Links: `Documentation <https://borgbackup.readthedocs.io/en/1.2-maint/>`_ · `Installation <https://borgbackup.readthedocs.io/en/1.2-maint/installation.html>`_ · `Downloads <https://github.com/borgbackup/borg/releases/latest>`_
+:Date: 2022-02-22
+
+This marks the first stable release in the Borg 1.2 series. Many improvements and new features
+were incorporated into Borg 1.2 since the 1.1 release of Borg in early 2017. Even more changes
+were made "under the hood": cleanups and refactorings.
+
+Many people contributed to this release — see the AUTHORS file in the git repo and the
+git history for details. Even more people contributed otherwise, e.g. by giving feedback
+or helped with testing.
```

Comment:
```suggestion
or helping with testing.
```

Reply:
i even thought about that, but obviously chose the wrong one. :-)